dipy/workflows/segment.py

```diff
@@ -1,84 +1,85 @@
 from __future__ import division, print_function, absolute_import
 
-from glob import glob
-from os.path import join, basename, splitext
+import inspect
+import logging
+import numpy as np
 
 import nibabel as nib
-import numpy as np
 
-from dipy.workflows.utils import choose_create_out_dir
 from dipy.segment.mask import median_otsu
+from dipy.workflows.workflow import Workflow
+
+
+class MedianOtsuFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'medotsu'
```

Comment:
They are used in combined workflow when generating all the available sub workflows parameters. They are added as prefixes to avoid having confusion when two sub workflows parameters share the same name. 
For example:
![sub_params_short_name](https://cloud.githubusercontent.com/assets/1280738/17300586/020e62b2-57e1-11e6-9a32-0db63a2ab37c.png)

you can see --**dti**.b0_threshold,  --**csd**.b0_threshold and --**csf**.b0_threshold

If none is defined, the classname is used (which is not very user friendly hence the addition of short names as an option for the person who creates a workflow).